tutorials/tutorial10/tutorial.ipynb

```diff
@@ -0,0 +1,412 @@
+    "                real=sol_train[sample_number].extract('u'))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, as the time progresses the solution becomes chaotic, which makes\n",
+    "it really hard to learn! We will now focus on building a Neural Operator using the\n",
+    "`SupervisedSolver` class to tackle the problem.\n",
+    "\n",
+    "## Averaging Neural Operator\n",
+    "\n",
+    "We will build a neural operator $\\texttt{NO}$ which takes the solution at time $t=0$ for any $x\\in\\Omega$,\n",
+    "the time $(t)$ at which we want to compute the solution, and gives back the solution to the KS equation $u(x, t)$, mathematically:\n",
+    "$$\n",
+    "\\texttt{NO}_\\theta : \\mathbb{U} \\rightarrow  \\mathbb{U},\n",
+    "$$\n",
+    "such that\n",
+    "$$\n",
+    "\\texttt{NO}_\\theta[u(t=0)](x, t) \\rightarrow  u(x, t).\n",
+    "$$\n",
+    "\n",
+    "There are many ways on approximating the following operator, e.g. by 2D [FNO](https://mathlab.github.io/PINA/_rst/models/fno.html) (for regular meshes),\n",
+    "a [DeepOnet](https://mathlab.github.io/PINA/_rst/models/deeponet.html), [Continuous Convolutional Neural Operator](https://mathlab.github.io/PINA/_rst/layers/convolution.html),\n",
+    "[MIONet](https://mathlab.github.io/PINA/_rst/models/mionet.html). \n",
+    "In this tutorial we will use the *Averaging Neural Operator* presented in [*The Nonlocal Neural Operator: Universal Approximation*](https://arxiv.org/abs/2304.13221)\n",
+    "which is a [Kernel Neural Operator](https://mathlab.github.io/PINA/_rst/models/base_no.html) with integral kernel:\n",
+    "\n",
+    "$$\n",
+    "K(v) = \\sigma\\left(Wv(x) + b + \\frac{1}{|\\Omega|}\\int_\\Omega v(y)dy\\right)\n",
+    "$$\n",
+    "\n",
+    "where:\n",
+    "\n",
+    "*   $v(x)\\in\\mathbb{R}^{\\rm{emb}}$ is the update for a function $v$ with $\\mathbb{R}^{\\rm{emb}}$ the embedding (hidden) size\n",
+    "*   $\\sigma$ is a non-linear activation\n",
+    "*   $W\\in\\mathbb{R}^{\\rm{emb}\\times\\rm{emb}}$ is a tunable matrix.\n",
+    "*   $b\\in\\mathbb{R}^{\\rm{emb}}$ is a tunable bias.\n",
+    "\n",
+    "If PINA many Kernel Neural Operators are already implemented, and the modular componets of the [Kernel Neural Operator](https://mathlab.github.io/PINA/_rst/models/base_no.html) class permits to create new ones by composing base kernel layers.\n",
+    "\n",
+    "**Note:*** We will use the already built class* `AveragingNeuralOperator`, *as constructive excercise try to use the* [KernelNeuralOperator](https://mathlab.github.io/PINA/_rst/models/base_no.html) *class for building a kernel neural operator from scratch. You might employ the different layers that we have in pina, e.g.* [FeedForward](https://mathlab.github.io/PINA/_rst/models/fnn.html), *and* [AveragingNeuralOperator](https://mathlab.github.io/PINA/_rst/layers/avno_layer.html) *layers*."
```

Comment:
Two typos: start of line 160 should be "# Note: *" (a space is missing). The AveragingNeuralOperator at the end of the is probably a typo for DeepONet/MiONet.

Reply:
Thanks! I think is a problem of the rending of GitHub, on local the `.ipynb` looks fine with the spacing.